bridge/contracts/HecBridgeSplitter.sol

```diff
@@ -170,9 +170,9 @@ contract HecBridgeSplitter is OwnableUpgradeable, PausableUpgradeable {
 			bridgeFees += bridgeFee;
 		}
 
-		if (sendingAsset != address(0)) {
-			if (msg.value < bridgeFees) revert INVALID_FEES();
+		if (msg.value < bridgeFees) revert INVALID_FEES();
```

Comment:
There's no need to consider the `feeAmounts` when sendingAsset == address(0)

Reply:
so when sendingAsset == address(0), the `msg.value` should be greater than the `bridgeFees + feeAmounts`, shouldn't it?